README.md

```diff
@@ -1 +1,67 @@
-# user-management-daemon
\ No newline at end of file
+# User Management Daemon
+
+Daemon for handling Sailfish OS device users.
+
+## Brief Description:
+
+
+This daemon provides D-Bus API for adding, removing and editing additional
+Sailfish OS device users. The main system user is created during the first boot
+of the device. This user can the use this API to add additional users which do
+only have access to some of the available functionality and services.
+
+## API:
+
+Below is the D-Bus interface introspection:
+
+`
+    <!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
```

Comment:
I suggest to have a link to the actual file, otherwise this comment will have to be updated every time the spec changes and sooner or later someone will forget to do that.